### PR TITLE
HOTT-1724: Add support for n/a on duty rate.

### PR DESCRIPTION
**Pull request by alexdesi**
### What?
Add support for n/a for specific combinations of geographical areas and measure types.

### Jira link
https://transformuk.atlassian.net/browse/HOTT-1724

### CODE REVIEW

File: spec/factories/duty_expression_factory.rb [hunk 1]
```diff
@@ -12,5 +12,13 @@
...
       description { 'VAT' }
     end
+
+    trait :with_geographical_area do
+      attributes_for { :geographical_area }
+    end
+
+    trait :with_measure_type do
+      attributes_for { :measure_type }
+    end
```
Comment (alexdesi) [inline spec/factories/duty_expression_factory.rb]: these can be removed

File: app/helpers/measures_helper.rb [hunk 2]
```diff
@@ -1,8 +1,11 @@
 module MeasuresHelper
   def filter_duty_expression(measure)
+    # NIHIL is an abbreviation (see backend DutyExpressionDescription model),
+    # our best guess is that the it is used to indicate that a previously applied measure
```
Comment (willfish) [inline app/helpers/measures_helper.rb]:
```suggestion
    # our best guess is that it is used to indicate that a previously applied measure
```

File: app/models/measure.rb [hunk 3]
```diff
@@ -179,6 +179,12 @@ def measure_conditions_with_guidance
     measure_conditions.select(&:has_guidance?)
   end
 
+  def measure_type_duty_label
+    return unless geographical_area.id && measure_type.id
+
+    I18n.t("measure_type_duties.#{geographical_area.id}.#{measure_type.id}", default: nil)
```
Comment (willfish) [inline app/models/measure.rb:183]:
```suggestion
    I18n.t("measure_type_duties.#{geographical_area&.id}.#{measure_type&.id}", default: nil)
```

You can drop the guard above with this syntax. I honestly don't think it's necessary in real life, though. Possibly you added it for the tests

File: app/models/measure.rb [hunk 4]
```diff
@@ -179,6 +179,12 @@ def measure_conditions_with_guidance
     measure_conditions.select(&:has_guidance?)
   end
 
+  def measure_type_duty_label
+    return unless geographical_area.id && measure_type.id
```
Comment (willfish) [inline app/models/measure.rb]: I think we should remove this

Comment (alexdesi) [reply app/models/measure.rb]: yes, I've checked, and measures always have geo ID and measure type ID

**Comment by alexdesi** [comment]
Screenshot 2022-07-07 at 14 55 23

**Review by willfish — APPROVED** [review]

**Pull request merged**